test/Project.toml

```diff
@@ -10,6 +10,7 @@ LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearMaps = "7a12625a-238d-50fd-b39a-03d52299707e"
 LinearOperators = "5c8ed15e-5a4c-59e4-a42b-c7e8811fb125"
+Match = "7eb4fadd-790c-5f42-8a69-bfa0b872bfbf"
```

Comment:
We're using MLStyle's `@match` now, so we don't want this dependency on Match (see #214).